global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "2.2.106"
+    "version": "2.2.402"

```

Comment:
Wonder if we should just go to 3.x ? Presumably you only went this far to avoid complicating this PR, and I think that's probably a good idea?

Reply:
Yes, I started and it got too much fast